pkg/utils/transform/transform.go

```diff
@@ -65,11 +65,44 @@ func InjectNamespaceConditional(preserveNamespace, targetNamespace string) mf.Tr
 		if ok && val == "true" {
 			return nil
 		}
-
 		return tf(u)
 	}
 }
 
+func InjectNamespaceRoleBinding(targetNamespace string) mf.Transformer {
+	return func(u *unstructured.Unstructured) error {
+		switch strings.ToLower(u.GetKind()) {
+		case "rolebinding":
+			subjects, _, _ := unstructured.NestedFieldNoCopy(u.Object, "subjects")
+			for _, subject := range subjects.([]interface{}) {
+				m := subject.(map[string]interface{})
+				if _, ok := m["namespace"]; ok {
+					m["namespace"] = targetNamespace
+				}
+			}
+			return nil
+		}
+		return nil
+	}
+}
+
+func InjectNamespaceWebhookClientConfig(targetNamespace string) mf.Transformer {
+	return func(u *unstructured.Unstructured) error {
+		switch strings.ToLower(u.GetKind()) {
+		case "customresourcedefinition":
+			service, _, _ := unstructured.NestedFieldNoCopy(u.Object, "spec", "conversion", "webhookClientConfig", "service")
+			if service != nil {
```

Comment:
this can be made flat by 
```
  if service == nil {
     return nil  
  }
```

but shouldn't you make use of the `found` return value ? 

```go

service, found, err := unstructured.NestedFieldNoCopy(...)
if !found || err != nil {
  return nil ??
}
```

Reply:
Yes, it makes sense, I have updated both transform functions accordingly. Thanks